addons/edi/tests/common.py

```diff
@@ -191,3 +187,16 @@ def assertRaisesIssue(self, entity, exception=UserError):
 
         # Delete the raised issue
         new_issue_ids.unlink()
+    
+    @classmethod
+    def _set_test_file_directory(cls, path):
+        """
+        Setup a new test file directory. This is not implemented as a class method
```

Comment:
It is implemented as a `classmethod` though?  Also there is no `TestEdiInterfacesBase` class in this repository.

Reply:
This is because I copy pasted the method from where it was before in `udes_interfaces`, I will update it my bad

Reply:
changed https://github.com/unipartdigital/odoo-edi/pull/44/commits/ec6bdaf32efa3251625f10fda82c3c1140017143